### DISCUSSION TITLE
How does the search in Quasar docs work?

### DISCUSSION THREAD

**Discussion by danbars** [quasarframework/quasar#12641] [General - Components / Directives / etc] [answered]
I'm building a documentation site that is built on Quasar's own docs.
In `/docs/package.json` there's a script that builds `/dist/indices.json` which is a JSON file with basically the entire content organized as json.
It seems that this file is used by a server that is not part of this repo. 
When you're searching the docs there's a `POST` request being made to `https://search.quasar.dev/indexes/quasar-v2/search`

Can someone please tell what is the server that runs the search? Is it based on an open source that I can also use? 
Thanks!

**Comment by metalsadman** [comment, marked as answer]
it uses https://www.meilisearch.com/

**Reply by danbars** [reply]
Thanks 👍
